lib/test_formulae.rb

```diff
@@ -97,33 +105,51 @@ def download_artifact_from_previous_run!(name)
         response = GitHub::API.open_rest(url)
         return if response.fetch("total_count").zero?
 
-        wanted_artifact = response.fetch("artifacts").find { |artifact| artifact.fetch("name") == name }
+        artifacts = response.fetch("artifacts")
+        wanted_artifact = artifacts.find { |artifact| artifact.fetch("name") == artifact_name }
+        # If we didn't find the artifact that we wanted, fall back to the `event_payload` artifact.
+        wanted_artifact ||= artifacts.find { |artifact| artifact.fetch("name") == "event_payload" }
```

Comment:
This logic now needs adjusting to account for the fact that `event_payload` will now be from `triage.yml`.